.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    branches: # Snapshots
```

Comment:
this will publish snapshots, which is what we want, but we will need to add 

```scala
// We are publishing snapshots to Sonatype
ThisBuild / dynverSonatypeSnapshots := true
```

To the build.sbt

Reply:
@@octoate In fact, this is not necessary. 😉 
Because If we are using `sbt-ci-release` for publishing (for Twirl we do, as I see), it already has this property.
https://github.com/sbt/sbt-ci-release/blob/main/plugin/src/main/scala/com/geirsson/CiReleasePlugin.scala#L105
It's my mistake, that I added this property to `play-ws` explicitly 😞

Reply:
Thanks. 
Do we only need the configuration for the main branch to publish as a SNAPSHOT version?
The travis settings include `/^\d+\.\d+(\.\d+)?(-\S*)?$/`, but I removed it because it doesn't match the branch that is currently in the twirl repository.

Reply:
@ihostage, you are right. No need to add it. I forgot about it. 

@tototoshi, I think as it is in the PR is fine.

I think we have everything except the credentials to publish.